plugins/scio/src/main/scala/org/broadinstitute/monster/sbt/MonsterScioPipelinePlugin.scala

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.monster.sbt
+
+import com.google.cloud.storage.{BlobInfo, StorageOptions}
+import sbt._
+import sbt.Keys._
+import sbtassembly.{AssemblyKeys, AssemblyPlugin}
+
+/** Plugin for projects which build a Scio ETL pipeline. */
+object MonsterScioPipelinePlugin extends AutoPlugin with AssemblyKeys {
+  override def requires: Plugins = MonsterBasePlugin && AssemblyPlugin
+
+  object autoImport extends MonsterScioPipelineKeys
+  import autoImport._
+
+  val ScioUtilsVersion = "1.0.0"
+
+  override def projectSettings: Seq[Def.Setting[_]] = Seq(
+    // Set best-practice compiler flags for Scio.
+    scalacOptions ++= Seq(
+      "-Xmacro-settings:show-coder-fallback=true",
+      "-language:higherKinds"
+    ),
+    // Add our common utils library.
+    // TODO: Are there any common test libraries that we could/should
+    // also add here?
+    libraryDependencies += "org.broadinstitute.monster" %% "scio-utils" % ScioUtilsVersion,
```

Comment:
Here up are the boilerplate options I saw in clinvar-ingest. Below this is new JAR-publishing logic.